docs/references/javascript/clerk/sign-up.mdx

```diff
@@ -1,13 +1,17 @@
 ---
-title: <SignUp /> component
-description: The <SignUp /> component renders a UI for signing up users. The functionality of the <SignUp /> component are controlled by the instance settings you specify in your Clerk Dashboard. You can further customize your SignUp component by passing additional properties at the time of rendering.
+title: JavaScript <SignUp />

```

Comment:
or maybe "<SignUp /> JS Reference" or "Use the <SignUp /> component with JavaScript" or something

Reply:
This is a good callout, and not an easy issue to solve. "JavaScript <SignUp>" makes sense for sure. I also like your suggestion of "Use the component with JavaScript" but I don't think it suits the page, which is a reference doc, so it's tough.

In my opinion we should lean toward something like "SignUp JavaScript Reference". The [page for the React component](https://docs-preview-764.clerkpreview.com/docs/components/authentication/sign-up) could be called "<SignUp> Component Reference", so it's more obvious from search.

I think the titles for the JS reference stuff should look less like a component, because even though they _are_ components, people think "React" when they see that word, and we should lean toward comprehensibility over accuracy in this case

Reply:
Yeah agreed, I don't think we use the component syntax here since it's not actually used. Something like "Sign Up Component - JavaScript SDK" feels reasonable to me 🤷‍♂️

Reply:
https://clerkinc.slack.com/archives/CHZ1FBBEG/p1709667576078549